deeplearning4j-core/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java

```diff
@@ -92,6 +93,7 @@ public MultiLayerNetwork(MultiLayerConfiguration conf) {
      * @param conf   the configuration json
      * @param params the parameters
      */
+    @Deprecated
```

Comment:
I marked these deprecated because they offer no opportunity to add a listener.   Listeners must be added before init().